tests/integration/individual_response/test_individual_response.py

```diff
@@ -79,6 +80,19 @@ def _request_individual_response(self):
         self.post()
 
 
+class TestIndividualResponseHubDisabled(IndividualResponseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey(
+            "test_individual_response_on_hub_disabled", region_code="GB-ENG"
+        )
+
+    def test_show_on_hub_false(self):
+        self._add_household_no_primary()
+
+        self.assertIsNone(self.individual_response_link)
```

Comment:
I think we should also assert that you are on the hub, as this would pass for any other page also.